cms/djangoapps/contentstore/views/certificates.py

```diff
@@ -395,98 +398,160 @@ def certificate_activation_handler(request, course_key_string):
     return HttpResponse(status=200)
 
 
-@login_required
-@require_http_methods(("GET", "POST"))
-@ensure_csrf_cookie
-def certificates_list_handler(request, course_key_string):
+class CertificatesListView(View):
```

Comment:
Use `APIView` for this.

Reply:
> Use `APIView` for this.

This View is responsible for handling legacy CMS certificates, that is based on Django templates. So for HTML page rendering we should use `View` instead of `APIView`.